docs/api/orchestrator_openapi_0_1_0.yaml

```diff
@@ -2,11 +2,18 @@ openapi: 3.0.1
 info:
   title: FMS Orchestrator API
   version: 0.1.0
+tags:
+    - name: Text Generation with detections
+      description: Detections on text generation model input and/or output
```

Comment:
nit: Can we specify "tasks" more explicitely here? 

Something like: `Task - Generation <br /> Description - Detections on text generation model input and/or output`

Reply:
not quite, but this works as well. Thanks